packages/open-payments/src/generated/types.ts

```diff
@@ -0,0 +1,702 @@
+/**
```

Comment:
@wilsonianb what we can do is instead of that check is actually just run the script on a pre-commit hook to generate the types and stage the updated file for committing. 

Instead of doing it on every commit, we can just do that _only_ if the `src/types.ts` file has been changed, for example. Thoughts?

Reply:
That sounds good to me.
We did recently remove pre-commit hooks, but hopefully it won't be an issue to support this
- https://github.com/interledger/rafiki/pull/545